Files/MultilingualResources/Files.ru-RU.xlf

```diff
@@ -3113,19 +3113,19 @@
         </trans-unit>
         <trans-unit id="BaseLayoutItemContextFlyoutPinToFavorites.Text" translate="yes" xml:space="preserve">
           <source>Pin to Favorites</source>
-          <target state="needs-review-translation" state-qualifier="tm-suggestion">Закрепить в избранном</target>
+          <target state="needs-review-translation" state-qualifier="tm-suggestion">Добавить в Избранное</target>
         </trans-unit>
         <trans-unit id="SideBarUnpinFromFavorites.Text" translate="yes" xml:space="preserve">
           <source>Unpin from Favorites</source>
-          <target state="new">Unpin from Favorites</target>
+          <target state="new">Убрать из Избранного</target>
```

Comment:
This state should be "needs-review-translation" or "translated", regarding the state I said on my previous comment (needs review for strings that should be reviewed in the future, translated for strings that are finished).

I suppose this will be:

```suggestion
          <target state="translated">Убрать из Избранного</target>
```

Apply this to all the strings that have the "new" state (lines 3124, 3236, 3244, 3260, 3264)